Makefile

```diff
@@ -45,8 +45,6 @@ deploy: azd-login ## 🚀 Deploy everything to Azure
 	@azd deploy function --no-prompt
 	@azd deploy adminweb --no-prompt
 
-docker-build: ## 🐳 Build the Docker image
```

Comment:
Don't we need this anymore?

Reply:
Github Actions carries out the build rather than locally - this was me testing locally. We don't actually "need" it so removed it